check-load/lib/check_load_netbsd.go

```diff
@@ -0,0 +1,31 @@
+package checkload
+
+import (
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+func getloadavg() (loadavgs [3]float64, err error) {
+	outputBytes, err := exec.Command("sysctl", "-n", "vm.loadavg").Output()
```

Comment:
Using `unix.SysctlRaw` is better if possible, to avoid overhead of executing command.
(ref: https://github.com/mackerelio/go-osstat/blob/master/loadavg/loadavg_bsd_nocgo.go )